storage/netapp_e/na_eseries_facts.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/python
+
+# (c) 2016, NetApp, Inc
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import json
+
+from ansible.module_utils.api import basic_auth_argument_spec
+from ansible.module_utils.basic import AnsibleModule, get_exception
+from ansible.module_utils.urls import open_url
+
+try:
+    from urllib.error import HTTPError
+except ImportError:
+    from urllib2 import HTTPError
```

Comment:
It's something in my editor that isn't configured correctly.  From the command line, it looks good.  Running 2.2.0 btw.

I have a test to run through the http exception handler and it's working as expeced.
```
10.251.230.29 | FAILED! => {
    "changed": false, 
    "failed": true, 
    "msg": "Error accessing storage-system with id [ansible4]. Error [<urlopen error [Errno 113] No route to host>]"
}
```